cockroachcloud/quickstart-trial-cluster.md

```diff
@@ -0,0 +1,183 @@
+3. Verify that the `us-west2 GCP` region and `default_db` database are selected.
+4. Click **Continue**. The **Connect** tab is displayed.
+5. Click **Connection string** to get the connection string for your cluster.
+6. Create a `certs` directory on your local workstation.
+7. Click the name of the `ca.crt` file to download the CA certificate to your local machine.
+8. Move the downloaded `ca.crt` file to the `certs` directory.
+
+## Step 5. Run your first query
+
+For this tutorial, we will use the [`movr` workload](../stable/movr.html) to run the first query. On your local machine:
+
+1. [Download the CockroachDB binary](../stable/install-cockroachdb.html):
+
+    <div class="filters clearfix">
+      <button class="filter-button page-level" data-scope="mac">Mac</button>
+      <button class="filter-button page-level" data-scope="linux">Linux</button>
+    </div>
+
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz \
+    | tar -xJ
+    ~~~
+    </section>
+
+    <section class="filter-content" markdown="1" data-scope="linux">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ wget -qO- https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.linux-amd64.tgz \
+    | tar  xvz
+    ~~~
+    </section>
+
+2. Copy the binary into the `PATH` so it's easy to run the SQL client from any location:
+
+    <div class="filters clearfix">
+      <button class="filter-button page-level" data-scope="mac">Mac</button>
+      <button class="filter-button page-level" data-scope="linux">Linux</button>
+    </div>
+
+    <section class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cp -i cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach /usr/local/bin/
+    ~~~
+    </section>
+
+    <section class="filter-content" markdown="1" data-scope="linux">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo cp -i cockroach-{{ page.release_info.version }}.linux-amd64/cockroach /usr/local/bin/
+    ~~~
+    </section>
+
+3. Initialize the `movr` workload using the `cockroach workload` command and the [connection string](#step-4-get-the-connection-string).
+
+    In the connection string, the SQL user's username is prepopulated. Replace `<password>` with the SQL user's password that you entered in [Step 2](#step-2-create-a-sql-user). Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 4](#step-4-get-the-connection-string).
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach workload init movr \
+    'postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    ~~~
+
+4. Use the built-in SQL client to view the database:
```

Comment:
could link to `cockroach sql` docs so users can learn about more things they can do